usaspending_api/agencies/management/commands/load_agencies.py

```diff
@@ -0,0 +1,276 @@
+        self.agency_file = options["agency_file"]
+        logger.info("AGENCY FILE: {}".format(self.agency_file))
+
+        self.force = options["force"]
+        logger.info("FORCE SWITCH PROVIDED: {}".format(self.force))
+        logger.info("MAX CHANGE LIMIT: {}".format("{:,}".format(MAX_CHANGES) if self.force else "unlimited"))
+
+        self.rows_affected_count = 0
+
+        try:
+            with Timer("Import agencies"):
+                with transaction.atomic():
+                    self._run_steps()
+        except Exception:
+            logger.error("ALL CHANGES WERE ROLLED BACK DUE TO EXCEPTION")
+            raise
+
+        try:
+            with Timer("Vacuum tables"):
+                self._vacuum_tables()
+        except Exception:
+            logger.error("CHANGES WERE SUCCESSFULLY COMMITTED EVEN THOUGH VACUUMS FAILED")
+            raise
+
+    def _execute_sql(self, sql):
+        with self.connection.cursor() as cursor:
+            cursor.execute(sql)
+            if cursor.rowcount > -1:
+                logger.info("{:,} rows affected".format(cursor.rowcount))
+        return cursor.rowcount
+
+    def _execute_sql_file(self, filename):
+        file = (SQL_PATH / filename).with_suffix(".sql")
+        sql = file.read_text()
+        return self._execute_sql(sql)
+
+    def _read_raw_agencies_csv(self):
+        agencies = read_csv_file_as_list_of_dictionaries(self.agency_file)
+        if len(agencies) < 1:
+            raise RuntimeError("Agency file '{}' appears to be empty".format(self.agency_file))
+
+        self.agencies = [
+            Agency(
+                row_number=row_number + 1,
+                cgac_agency_code=prep(agency["CGAC AGENCY CODE"]),
+                agency_name=prep(agency["AGENCY NAME"]),
+                agency_abbreviation=prep(agency["AGENCY ABBREVIATION"]),
+                frec=prep(agency["FREC"]),
+                frec_entity_description=prep(agency["FREC Entity Description"]),
+                frec_abbreviation=prep(agency["FREC ABBREVIATION"]),
+                subtier_code=prep(agency["SUBTIER CODE"]),
+                subtier_name=prep(agency["SUBTIER NAME"]),
+                subtier_abbreviation=prep(agency["SUBTIER ABBREVIATION"]),
+                toptier_flag=bool(strtobool(prep(agency["TOPTIER_FLAG"]))),
+                is_frec=bool(strtobool(prep(agency["IS_FREC"]))),
+                user_selectable=bool(strtobool(prep(agency["USER SELECTABLE ON USASPENDING.GOV"]))),
+                mission=prep(agency["MISSION"]),
+                website=prep(agency["WEBSITE"]),
+                congressional_justification=prep(agency["CONGRESSIONAL JUSTIFICATION"]),
+                icon_filename=prep(agency["ICON FILENAME"]),
+                include_toptier_without_subtier=prep(agency["CGAC AGENCY CODE"]) in DOD_ARMED_FORCES_CGAC,
+            )
+            for row_number, agency in enumerate(agencies)
+        ]
+
+    def _validate_raw_agencies(self):
```

Comment:
I saw this was an issue in codeclimate; You can make this function just validate one agency, then run that in a loop, and that will have the Cognitive Complexity